src/coupling/application.cc

```diff
@@ -501,7 +507,8 @@ void Application::init(int argc, char ** argv) {
     Profiler::instance()->set_program_info("Flow123d",
             rev_num_data.version, rev_num_data.branch, rev_num_data.revision, build);
 
-    //Profiler::instance();
+    if (use_profiler & memory_monitoring)
+        Profiler::set_memory_monitoring(true);
```

Comment:
Rather: `Profiler::set_memory_monitoring(memory_monitoring)`, no problem to set it even if use_profiler is false.